src/main/java/io/github/mooy1/slimechem/utils/SuperNum.java

```diff
@@ -5,43 +5,34 @@
 /**
  * Superscript numbers and getter
  * 
+ * @author TheBusyBiscuit
  * @author Mooy1
  * 
  */
 public final class SuperNum {
-    
-    //I tried unicode but it messes up
-    public static final String ZERO = "⁰"; 
-    public static final String ONE = "¹";
-    public static final String TWO = "²";
-    public static final String THREE = "³";
-    public static final String FOUR = "⁴";
-    public static final String FIVE = "⁵";
-    public static final String SIX = "⁶";
-    public static final String SEVEN = "⁷";
-    public static final String EIGHT = "⁸";
-    public static final String NINE = "⁹";
-    
-    
+
+    // I tried unicode but it messes up
+    public static final char ZERO_STARTING_POINT = '⁰';
+
+    private SuperNum() {}
+
     @Nonnull
     public static String fromInt(int i) {
         StringBuilder builder = new StringBuilder();
-        
-        for (char c : String.valueOf(i).toCharArray()) {
-            switch (c) {
-                case '0': builder.append(ZERO); break;
-                case '1': builder.append(ONE); break;
-                case '2': builder.append(TWO); break;
-                case '3': builder.append(THREE); break;
-                case '4': builder.append(FOUR); break;
-                case '5': builder.append(FIVE); break;
-                case '6': builder.append(SIX); break;
-                case '7': builder.append(SEVEN); break;
-                case '8': builder.append(EIGHT); break;
-                case '9': builder.append(NINE); break;
-            }
+        int num = i;
+
+        while (num > 0) {
+            // Use zero char as a base and then add x onto it.
+            builder.append((char) (ZERO_STARTING_POINT + (num % 10)));
+            num /= 10;
         }
 
-        return builder.toString();
+        return builder.reverse().toString();
+    }
+
+    public static void main(String[] args) {
+        for (int i = 0; i < 1000; i++) {
+            System.out.println(fromInt(i));
+        }
```

Comment:
Why...

Reply:
whoops XD
Forgot to remove that test code